thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -122,38 +121,123 @@ body {
 
 .te-search-results {
   border-bottom: 1px app-shade(black, 1) solid;
+  color: app-shade(black, 0.7);
   padding: 24px 0;
 
   &__list {
     list-style: none;
-    // margin-bottom: 24px;
+    font-weight: normal;
+    margin-bottom: 0;
+
+    &--summary-block {
+      padding: 0;
+    }
 
     &--details-block {
       padding: 0;
     }
   }
 
+  &__header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: $te-spacing;
+  }
+
+  &__row {
+    display: flex;
+    font-size: 15px;
+  }
+
+  &__option {
+    display: inline-block;
+    width: 100px;
+    min-width: 100px;
+    font-weight: 600;
+    font-size: 14px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    text-align: right;
+    margin-right: 10px;
+    color: app-shade(black, 4);
+  }
+
+  &__option-solo {
```

Comment:
this seems to be a modifier i.e `--solo`

Reply:
yes - fixed